nixos/modules/services/matrix/mautrix-meta.nix

```diff
@@ -0,0 +1,210 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
We shouldn't introduce new occurrences of with lib over entire files.

Reply:
Agreed, implemented, thank you.